src/ray/object_manager/object_manager.cc

```diff
@@ -865,13 +869,22 @@ std::string ObjectManager::DebugString() const {
   return result.str();
 }
 
-void ObjectManager::RecordMetrics() const {
+void ObjectManager::RecordMetrics() {
+  pull_manager_->RecordMetrics();
+  push_manager_->RecordMetrics();
   stats::ObjectStoreAvailableMemory().Record(config_.object_store_memory - used_memory_);
   stats::ObjectStoreUsedMemory().Record(used_memory_);
   stats::ObjectStoreFallbackMemory().Record(
       plasma::plasma_store_runner->GetFallbackAllocated());
   stats::ObjectStoreLocalObjects().Record(local_objects_.size());
   stats::ObjectManagerPullRequests().Record(pull_manager_->NumActiveRequests());
+  ray::stats::STATS_num_chunks_received_total.Record(num_chunks_received_total_);
+  ray::stats::STATS_num_chunks_received_failed.Record(num_chunks_received_total_failed_,
+                                                      "Total");
+  ray::stats::STATS_num_chunks_received_failed.Record(num_chunks_received_cancelled_,
+                                                      "Cancelled");
+  ray::stats::STATS_num_chunks_received_failed.Record(
```

Comment:
Since these are new stats, can we consolidate them under a single `object_manager_chunks_received` metric with labels for breakdown?